presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftConnectorConfig.java

```diff
@@ -28,6 +28,7 @@
     private DataSize maxResponseSize = new DataSize(16, MEGABYTE);
     private int metadataRefreshThreads = 1;
     private int lookupRequestsConcurrency = 1;
+    private boolean setIdentityThriftHeaders;
```

Comment:
You might need to make corresponding changes in `TestThriftConnectorConfig` as well